getting-started/multiband-analysis.rst

```diff
@@ -94,6 +94,7 @@ Before going ahead and plotting a CMD from the full source table, you'll typical
 Exactly what filtering is done depends on the application, but source tables should *always* be filtered for unique sources.
 There are two ways that measured sources might not be unique: deblended sources and sources in patch overlaps.
 Additionally, some sources are "sky" objects added by ``detectCoaddSources.py`` for noise characterization that you need to filter out.
+This section gives a brief introduction to removing duplicate and unwanted sources, but a more thorough description can be found in `this Community post <https://community.lsst.org/t/new-use-of-isprimary-that-began-in-w-2021-12/4957>`_.
```

Comment:
It's great that you wrote that Community post, but keep in mind that our documentation doesn't live on the Community forum. The forum links to pipelines.lsst.io, but not the other way around. I'd like us to delete this link to community.lsst.org and instead link to a page in the `lsst.pipe.tasks` documentation.

You can start by literally copying the content from the [forum post](https://community.lsst.org/t/new-use-of-isprimary-that-began-in-w-2021-12/4957) and adding a new rst file to `pipe_tasks` at `doc/lsst.pipe.tasks/deblending-flags-overview.rst` (and add that [file to the "Using lsst.pipe.tasks" toctree](https://developer.lsst.io/stack/module-homepage-topic-type.html#using-module-and-toctrees) in doc/lsst.pipe.tasks/index.rst).